include/llbuild/BuildSystem/BuildSystemFrontend.h

```diff
@@ -58,29 +58,13 @@ enum class CommandResult;
 ///
 /// NOTE: This class is *NOT* thread safe.
```

Comment:
Is this NOTE still true? Should any other relationships be updated?

Reply:
Hmm, I'll consider and update.